pyproject.toml

```diff
@@ -9,13 +9,17 @@ author-email = "thomas@kluyver.me.uk"
 home-page = "https://github.com/pypa/pep517"
 description-file = "README.rst"
 requires = [
-    "toml;python_version<'3.6'",
-    "tomli >=1.1.0;python_version>='3.6'",
+    "tomli >=1.1.0",
     "importlib_metadata;python_version<'3.8'",
     "zipp;python_version<'3.8'",
 ]
+requires-python = ">=3.6"
 classifiers = [
     "License :: OSI Approved :: MIT License",
-    "Programming Language :: Python :: 2.7",
     "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.6",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
```

Comment:
```suggestion
```

I deliberately leave these classifiers out. The `requires-python` specifies the same thing in a more precise and useful way (useful because pip respects it), and doesn't have to be updated when a new Python version comes out.

Reply:
No problem. I will remove it.

FWIW, some users use this information to determine the suitability of library within a particular project. It helps communicate  precise minor versions support and how up to date a release is. Further, the trove classifiers are easier to parse so it is useful for scripts.